testing/cases/phantomjs/mapper.js

```diff
@@ -128,6 +128,14 @@ describe('vgl.mapper', function () {
       mapper.geometryData().addSource(src2);
       mapper.modified();
       mapper.render(renderState);
+
+      /* Zero length indices shouldn't generate drawArray calls */
+      line.setIndices([]);
+      point.setIndices([]);
+      glCounts = $.extend({}, vgl.mockCounts());
+      mapper.modified();
+      mapper.render(renderState);
+      expect(vgl.mockCounts().drawArrays).toBe((glCounts.drawArrays || 0) + 3);
```

Comment:
We had been rendering five different items in the test (line 117).  I set two of the tests to have zero-length buffers, so three will still cause `drawArrays` to be called.